client.py

```diff
@@ -96,14 +96,15 @@ def server_connect():
             time.sleep(5)  # wait 5 seconds to try again
         else: break
 
-    strUserInfo = f"{socket.gethostname()}`,{platform.system()} {platform.release()}"
+    arrUserInfo = [socket.gethostname()]
+    strPlatform = f"{platform.system()} {platform.release()}"

```

Comment:
Add a check for if it returns emtpy.

Reply:
When platform is not able to retrieve info, it will return an empty string, which would not break the json code. The problem has to be something else

Reply:
Im not sure then, im not able to reproduce the error.